datafusion/functions/src/regex/regexpreplace.rs

```diff
@@ -401,8 +401,7 @@ fn _regexp_replace_static_pattern_replace<T: OffsetSizeTrait>(
         DataType::Utf8View => {
             let string_view_array = as_string_view_array(&args[0])?;
 
-            let mut builder = StringViewBuilder::with_capacity(string_view_array.len())
-                .with_block_size(1024 * 1024 * 2);
+            let mut builder = StringViewBuilder::with_capacity(string_view_array.len());
```

Comment:
does that mean the block size is not used in the builder for this case?

Reply:
yes, we don't need it after this pr is merged: https://github.com/apache/arrow-rs/pull/6136
I ran a local benchmark and show no perf diff